modAionImpl/src/org/aion/zero/impl/cli/Arguments.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ */
+package org.aion.zero.impl.cli;
+
+import java.util.ArrayList;
+import java.util.List;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+/**
+ * Command line arguments for the Aion kernel.
+ *
+ * @author Alexandra Roatis
+ */
+@Command(name = "./aion.sh", separator = " ", sortOptions = false, abbreviateSynopsis = true)
+public class Arguments {
+
+    // usage information
+    @Option(
+            names = {"-h", "--help"},
+            usageHelp = true,
+            description = "display help information")
+    private boolean help = false;
+
+    // account management
+    @Option(
+            names = {"ac", "-a create", "--account create"},
+            description = "create a new account")
+    private boolean createAccount = false;
+
+    @Option(
+            names = {"al", "-a list", "--account list"},
+            description = "list all existing accounts")
+    private boolean listAccounts = false;
+
+    @Option(
+            names = {"ae", "-a export", "--account export"},
+            paramLabel = "<account>",
+            description = "export private key of an account")
+    private String exportAccount = null;
+
+    @Option(
+            names = {"ai", "-a import", "--account import"},
+            paramLabel = "<key>",
+            description = "import private key")
+    private String importAccount = null;
+
+    // config generation
+    @Option(
+            names = {"-c", "--config"},
+            arity = "0..1",
+            paramLabel = "<network>",
+            description =
+                    "create config for the selected network\noptions: mainnet, conquest, mastery")
+    private String config = null;
+
+    // get information and version
+    @Option(
+            names = {"-i", "--info"},
+            description = "display information")
+    private boolean info = false;
+
+    @Option(
+            names = {"-v"},
+            description = "display version")
+    private boolean version = false;
+
+    @Option(
+            names = {"--version"},
```

Comment:
Can we call this --versionTag?

Reply:
Oh, if there's pre-existing behaviour based on it, I guess we should keep it the unchanged then

Reply:
Most of applications use `--version`  to check the version info.
Therefore I prefer don't change it.